ehr/server/app/config/db.config.js

```diff
@@ -11,9 +11,9 @@ Modified:
 
 **************************************************************************/
 module.exports = {
-    HOST: "localhost",
-    USER: "",
-    PASSWORD: "",
-    DB: "",
-    dialect: "mysql"
-  };
\ No newline at end of file
+  HOST: "localhost",
+  USER: "root",
+  PASSWORD: "root",

```

Comment:
We never want to include sensitive information in files we add to GitHub, even if it is just credentials for local or development testing. Here, you would want to put something similar to:
Host: "ADD HOST HERE"
User: "ADD USER HERE"
Password: "ADD PASSWORD HERE"
DB: "ADD DB HERE"